docs/explanation/cryptography.rst

```diff
@@ -0,0 +1,303 @@
+.. _explanation_cryptographic-technology:
+
+Cryptographic technology in Snapcraft
+=====================================
+
+Snapcraft uses cryptographic technologies to fetch arbitrary files over the internet,
+communicate with local processes, and store credentials. It does not directly implement
+its own cryptography, but it does depend on external libraries to do so.
+
+When building snaps, Snapcraft uses different codebases and libraries depending on
+the :ref:`base snap <base-snaps>`. This means that the cryptographic techonology used
+depends on the base snap.
+
+core24 and newer
+----------------
+
+Snaps using core24 or newer as their base snap depend on Craft Application to build. For
```

Comment:
What do you think about:

```suggestion
Snaps using core24 or newer as their base depend on Craft Application to build. For
```

We could also link to the bases explanation.

Reply:
I have a link to the bases reference in a previous section. Do you think I should swap that out for the explanation?  

I'm not sure if I like the idea of nearby inline links to reference and explanation of the same subject.  I would be fine with a "For an explanation of base, see ..." link.

Reply:
Let's leave it as-is. The reference should be fine.